src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1208,4 +1211,11 @@ public static synchronized void updateBurns()
         FurnaceTileEntity.getBurnTimes().entrySet().forEach(e -> VANILLA_BURNS.put(e.getKey().delegate, e.getValue()));
     }
 
+	public static List<ItemStack> modifyLoot(List<ItemStack> list, LootContext context) {
+		LootModifierManager man = LootModifierManager.getInstance();
+		for(IGlobalLootModifier mod : man.getAllLootMods()) {
+			list = mod.apply(list, context);
```

Comment:
Do we want to run through each modifier in an arbitrary order?
It could lead to bugs where 1 mod converts wheat_seeds to wheat, and another converts wheat to seeds. depending on the order of registration / implementation of ImmutableMap, you could get different drops.
The easiest solution would be to run through them in alphabetical order, unless there is a need for something more specific.

Reply:
Arbitrary order is still better than complete overwriting. I'd be ok making it alphabetical (arbitrary, but deterministic).

Reply:
@Alpvax See [this chunk](https://github.com/MinecraftForge/MinecraftForge/pull/6267/files#diff-561423d31c6aff656e3a12b6de08223aR59) regarding deterministic sort, which should resolve this.